Whisky/Views/Bottle Views/BottleView.swift

```diff
@@ -131,6 +131,34 @@ struct BottleView: View {
                         showWinetricksSheet.toggle()
                     }
                 }
+                Button("button.addShortcut") {
+                    let panel = NSOpenPanel()
+                    panel.allowsMultipleSelection = false
+                    panel.canChooseDirectories = false
+                    panel.canChooseFiles = true
+                    panel.allowedContentTypes = [UTType.exe,
+                                                 UTType(importedAs: "com.microsoft.msi-installer")]
+                    panel.directoryURL = bottle.url.appendingPathComponent("drive_c")
+                    panel.begin { result in
+                        programLoading = true
+                        Task(priority: .userInitiated) {
+                            if result == .OK {
+                                if let url = panel.urls.first {
+                                    do {
+                                        let name = url.lastPathComponent
+                                        try await Wine.addShortcut(name: name, url: url, bottle: bottle)
```

Comment:
Validation for bottle shortcuts should be added. Removing or renaming the source file will currently not remove the dead shortcut, and will just log errors.

Reply:
I could add a check to updateStartMenu(), so shortcuts are verified upon updates. Would that be sufficient? I don't want to stray too far from your preferred coding guidelines.

Reply:
> Validation for bottle shortcuts should be added. Removing or renaming the source file will currently not remove the dead shortcut, and will just log errors.

I could add a check to updateStartMenu(), so shortcuts are verified upon updates. Would that be sufficient? I don't want to stray too far from your preferred coding guidelines.